lib/storage_tables/service/disk_service.rb

```diff
@@ -102,6 +102,30 @@ def restore(_checksum, *_args)
 
       private
 
+      def private_url(checksum, expires_in:, content_type:, disposition:, **)
```

Comment:
Hier je er ook voor kunnen kiezen om te raisen als er te veel argumenten worden meegegeven? Als we een use case vinden waarbij dit niet handig is, kunnen we er een configuration setting van maken.

```suggestion
      def private_url(checksum, expires_in:, content_type:, disposition:)
```